aws-iot-accountauditconfiguration/aws-iot-accountauditconfiguration.json

```diff
@@ -46,8 +46,47 @@
     "AuditCheckConfigurations": {
       "description": "Specifies which audit checks are enabled and disabled for this account.",
       "type": "object",
-      "patternProperties": {
-        "[A-Z_]+": {
+      "properties": {
+        "AuthenticatedCognitoRoleOverlyPermissiveCheck": {
+          "$ref": "#/definitions/AuditCheckConfiguration"
+        },
+        "CaCertificateExpiringCheck": {
+          "$ref": "#/definitions/AuditCheckConfiguration"
+        },
+        "CaCertificateKeyQualityCheck": {
+          "$ref": "#/definitions/AuditCheckConfiguration"
+        },
+        "ConflictingClientIdsCheck": {
+          "$ref": "#/definitions/AuditCheckConfiguration"
+        },
+        "DeviceCertificateExpiringCheck": {
+          "$ref": "#/definitions/AuditCheckConfiguration"
+        },
+        "DeviceCertificateKeyQualityCheck": {
+          "$ref": "#/definitions/AuditCheckConfiguration"
+        },
+        "DeviceCertificateSharedCheck": {
+          "$ref": "#/definitions/AuditCheckConfiguration"
+        },
+        "IotPolicyOverlyPermissiveCheck": {
+          "$ref": "#/definitions/AuditCheckConfiguration"
+        },
+        "IotRoleAliasAllowsAccessToUnusedServicesCheck": {
+          "$ref": "#/definitions/AuditCheckConfiguration"
+        },
+        "IotRoleAliasOverlyPermissiveCheck": {
+          "$ref": "#/definitions/AuditCheckConfiguration"
+        },
+        "LoggingDisabledCheck": {
+          "$ref": "#/definitions/AuditCheckConfiguration"
+        },
+        "RevokedCaCertificateStillActiveCheck": {
+          "$ref": "#/definitions/AuditCheckConfiguration"
+        },
+        "RevokedDeviceCertificateStillActiveCheck": {
+          "$ref": "#/definitions/AuditCheckConfiguration"
+        },
+        "UnauthenticatedCognitoRoleOverlyPermissiveCheck": {
```

Comment:
Can we add description to these fields? If yes, then let's add the check name/enum that customers pass in API calls. Although the names you have assigned are pretty straightforward but still it might be better to document what checks these fields map to.

Reply:
The description added here does not get populated in the docs. So there is no big advantage to adding the description here. Also, there is no precedence to adding "description" with "$ref" so risking breaking something. User more likely would be looking at the docs for description rather than in the code. Voting to not include description.